src/SecureStore.BeyondTrust/SecureStore.BeyondTrust.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+	<PropertyGroup>
+		<TargetFramework>netstandard2.0</TargetFramework>
+		<CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
```

Comment:
I would include this, to be more in line with other plugins:
```xml
    <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
    <AssemblyName>UiPath.Orchestrator.Extensions.SecureStores.BeyondTrust</AssemblyName>
    <RootNamespace>UiPath.Orchestrator.Extensions.SecureStores.BeyondTrust</RootNamespace>
```

Reply:
Fixed